src/cohortManager.js

```diff
@@ -0,0 +1,60 @@
+class CohortManager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  createCohort(cohortName) {
+    const existingCohort = this.cohorts.find(
+      (cohort) => cohort.name === cohortName
+    )
+
+    if (existingCohort) {
+      throw new Error('Cohort name already exists')
+    }
+    const cohort = { name: cohortName, students: [] }
+    this.cohorts.push(cohort)
+    return cohort
+  }
+
+  findCohort(cohortName) {
+    const existingCohort = this.cohorts.find(
+      (cohort) => cohort.name === cohortName
+    )
+
+    if (!existingCohort) {
+      throw new Error('Cohort does not exist')
+    }
+    return existingCohort
+  }
+
+  removeCohortByName(cohortName) {
+    this.cohorts = this.cohorts.filter((cohort) => cohort.name !== cohortName)
+    return 'removed successfully'
+  }
```

Comment:
Returning strings like this are typically difficult to work with for the dev using the function. A better approach here is either to return a boolean (false if it failed, true if it was successful), or return nothing at all if it was successful and throw an error if it fails.

You could also just filter the array like you are doing and return nothing